README.md

```diff
@@ -314,7 +314,6 @@ terraform {
   required_providers {
     iterative = {
       source = "iterative/iterative"
-      version = "0.5.1"
```

Comment:
Not sure if dropping this is best, but seems hard to keep it updated in the readme

Reply:
Yes!